code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -123,6 +141,14 @@
 	build_path = /obj/item/reagent_containers/glass/bottle/killer/pestkiller
 	category = list("initial","Botany Chemicals")
 
+/datum/design/plant_b_gone
```

Comment:
This can also be made pretty easily with water and toxin. But I'm not sure it matters that much. Still, make the build path a reagent and double the costs/results ratio.